flake.lock

```diff
@@ -654,11 +654,11 @@
       },
       "locked": {
         "dir": "nix",
-        "lastModified": 1720980234,
-        "narHash": "sha256-YburlHd9Y56pYrXTXqSJ1M0KhaxTgK5Anbsbb2cA9J8=",
+        "lastModified": 1721056185,
+        "narHash": "sha256-GbqgpLdc31HudFA5bXe270f+/uUXVpa3aZu+yywssC8=",
         "owner": "wez",
         "repo": "wezterm",
-        "rev": "b20c61926ea086cfca6a411e38b215c3045bb12c",
+        "rev": "c9116830c27baf0c547a1524d33363fd5e42295a",
```

Comment:
Triggered for https://github.com/kachick/dotfiles/issues/679#issuecomment-2229384776